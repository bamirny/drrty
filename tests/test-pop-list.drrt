def int main(){
	list[str] condiments;
	int len;
	int new_len;

	condiments = ["ketchup", "mustard", "mayo", "hot sauce"];
	len = length(condiments);
	print(len);

	pop(condiments);
	new_len = len -1;
	
	print(new_len);

	prints(condiments[new_len-1]);

	return 0;

}