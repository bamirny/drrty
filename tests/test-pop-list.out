4<br>
3<br>
<span>mayo</span><br>
